src/js/components/leftPanel/dataPanel/DataTabView.tsx

```diff
@@ -0,0 +1,127 @@
+    const activeLayerInfo = activeFeatures.find(f => f.layerID === activeLayer);
+
+    const LayerAttributesElement = (props: {
+      activeLayerInfo: any;
+    }): JSX.Element => {
+      const [page, setPage] = useState(0);
+
+      function turnAttributeTablePage(forward: boolean): void {
+        let newPage;
+        if (forward) {
+          newPage =
+            page === props.activeLayerInfo.features.length - 1
+              ? page
+              : page + 1;
+        } else {
+          newPage = page === 0 ? 0 : page - 1;
+        }
+        setPage(newPage);
+      }
+
+      interface AttributeObject {
+        [key: string]: string;
+      }
+      const AttributeTable = (props: AttributeObject): JSX.Element => {
+        return (
+          <table>
+            <tbody>
+              {Object.keys(props.attributes).map((a: string, i: number) => (
+                <tr key={i}>
+                  <td>{a}</td>
+                  <td>{props.attributes[a]}</td>
+                </tr>
+              ))}
+            </tbody>
+          </table>
+        );
+      };
+
+      //determine if next/prev buttons are enabled or disabled
+      const prevBtn = page === 0 ? 'disabled' : '';
+      const nextBtn =
+        page === props.activeLayerInfo.features.length - 1 ? 'disabled' : '';
+      return (
+        <div className="layer-feature-group">
+          <div className="layer-controls">
+            <LayerSelector
+              activeFeatures={activeFeatures}
+              activeLayerInfo={activeLayerInfo}
+              handleLayerSelection={(layerID: string): void =>
+                setActiveLayer(layerID)
+              }
+            />
+            <div className="attribute-page-buttons">
+              <button
+                className={`attribute-page-button ${prevBtn}`}
+                onClick={() => turnAttributeTablePage(false)}
+              >
+                Prev
+              </button>
+              <button
+                className={`attribute-page-button ${nextBtn}`}
+                onClick={() => turnAttributeTablePage(true)}
+              >
+                Next
+              </button>
+            </div>
+          </div>
+          <div className="page-numbers">
+            {page + 1} / {props.activeLayerInfo.features.length}
+          </div>
+          <div className="layer-title">{props.activeLayerInfo.layerTitle}</div>
+          <AttributeTable
+            attributes={props.activeLayerInfo.features[page].attributes}
+          />
+        </div>
+      );
+    };
+
+    //TODO: needs to be active language aware
+    return (
+      <div className="data-tabview-container">
+        <LayerAttributesElement activeLayerInfo={activeLayerInfo} />
+        <DataTabFooter />
+      </div>
+    );
+  };
+
+  const DataTabViewContent = () => {
+    const tabViewIsVisible = tabViewVisible && activeTab === props.label;
+    if (!tabViewIsVisible) {
+      return null;
```

Comment:
@vaidotasp Thoughts on whether this Component should be _ripped_ off of the DOM (and more importantly, have its local state of `3 of 14` or whatever be reset) when we have a different tab active (or no tab active), then come back to `1 of 14` every time that we switch tabs? This is Not a change request, let's look at this further down the line.

Reply:
Ooh let's also check this out again Once We Get The Highlights On The Map For The Selected Feature!